doc/assessment.md

```diff
@@ -19,93 +19,219 @@ Each assessment is a single directory in the `assessments` folder. The directory
+Each assessment has a `title` describing its topic. Additionally, assessments are organized into `sets` (e.g., `Homework`, `Quiz`, `Exam`) and within each set the assessment has a `number`. Depending on the context, assessments are referred to by either an _abbreviation_, a _short name_ or a _long name_. The format of these is:
+
+* Abbreviation = `[Set abbreviation][Number]` (e.g., `Q2` to mean the second quiz).
+
+* Short name = `[Set] [Number]` (e.g., `Quiz 2`).
+
+* Long name = `[Set] [Number]: [Title]` (e.g., `Quiz 2: Coordinates and Vectors`).
 
-There is currently no way to disable question order randomization for `Exam` assessments. However, the order of `zones` is fixed (see below), which can be used to control question order.
+The allowable set names are specified in the [`courseInfo.json`](course.md#assessment-sets) file.
 
-## Question specification
+The `type` of an assessment does not have to correspond to the `set` it is in, but these are generally compatible. For example, `”type”: “Homework”` assessments normally have `”set”: “Homework”` or `”set”: “Machine Problem”`. On the other hand, `”type”: “Exam”` assessments normally have `”set”: “Quiz”`, `”set”: “Exam”`, or similar.
+
+## `allowAccess`: Access control
+
+See the [Access control page](accessControl.md) for details.
 
-An assessment is broken down in to a list of zones, like this:
+By default, an assessment is only accessible to `Instructor` users. Add access rules to make it available to students.
+
+## `zones`: The list of questions on the assessment
+
+An assessment is broken down in to a list of zones, each of which contains a list of question slots, like this:
 
 ```json
 "zones": [
     {
         "title": "Easy questions",
         "questions": [
-            {"id": "anEasyQ", "points": [10, 5, 3, 1, 0.5, 0.25]},
-            {"id": "aSlightlyHarderQ", "points": [10, 9, 7, 5]}
+            {"id": "anEasyQ", "points": [10]},
+            {"id": "aSlightlyHarderQ", "points": [5, 2]}
         ]
     },
     {
         "title": "Hard questions",
         "questions": [
-            {"id": "hardQV1", "points": 10},
-            {"id": "reallyHardQ", "points": [10, 10, 10]},
-            {
-                "numberChoose": 1,
-                "points": 5,
-                "alternatives": [
-                    {"id": "FirstAltQ", "points": 10},
-                    {"id": "SecondAltQ"}
-                ]
-            }
-        ]
+            {"id": "hardQV1", "points": [5]},
+            {"id": "reallyHardQ", "points": [5, 5, 5]},
+            {“id”: “hardestQV3”, “points”: [3, 2, 1, 0.5, 0.2]}
```

Comment:
```suggestion
            {"id": "hardestQV3", "points": [3, 2, 1, 0.5, 0.2]}
```

I see more smart quotes elsewhere in this document; should probably do a find/replace on the whole thing.

Reply:
Thanks, I think I got them all now. “Helpful” editors!